tools/ci/init_gcloud

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-echo $GCLOUD_SERVICE_KEY > ${HOME}/gcloud-service-key.json
-gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
-gcloud --quiet config set project ${GOOGLE_PROJECT_ID}
-gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
+if [ -z "$GCLOUD_SERVICE_KEY" ]
+then
+  echo "Skipping gcloud initiation because no service key is set"
```

Comment:
Could we add an `exit 0` line after this? Potential source of bugs if someone appends to this script and it would get executed